tests/generators/test_jinja.py

```diff
@@ -1,25 +1,37 @@
+from __future__ import annotations
+
+from functools import partial
+from typing import Callable
 from unittest.mock import patch
 
 import pytest
 from dynamicprompts.generators.jinjagenerator import JinjaGenerator
 from dynamicprompts.generators.promptgenerator import GeneratorException
 from dynamicprompts.jinja_extensions import DYNAMICPROMPTS_FUNCTIONS
-from dynamicprompts.wildcardmanager import WildcardManager
+from dynamicprompts.parser.config import ParserConfig
+from dynamicprompts.wildcardmanager import WildcardManager, to_wildcard
 
 GET_ALL_VALUES = "dynamicprompts.wildcardmanager.WildcardManager.get_all_values"
 
 
 @pytest.fixture
-def generator(wildcard_manager: WildcardManager):
-    return JinjaGenerator(wildcard_manager)
+def to_ww(wildcard_manager: WildcardManager) -> Callable[[str], str]:
+    ww = wildcard_manager.wildcard_wrap
+    return partial(to_wildcard, wildcard_wrap=ww)
```

Comment:
Sounds like we'd just want a `wildcard_manager.to_wildcard()` method instead of having to do this..?